src/OrleansCodeGenerator/project.json

```diff
@@ -1,6 +1,6 @@
 {
   "dependencies": {
-    "Microsoft.CodeAnalysis.CSharp": "1.0.0"
+    "Microsoft.CodeAnalysis.CSharp": "1.3.2"
```

Comment:
Had to update it in order to run with latest <code>BenchmarkDotNet</code> package. cc @ReubenBond